distribution/docs/src/main/resources/_contents/_sources/catalog-stores-contents.adoc

```diff
@@ -0,0 +1,6 @@
+
+=== Included Catalog Stores
+
+The following catalog stores are included with a standard installation of ${branding}:
+
+<<_registry_store, Registry Store>>:: the interface that allows CSW messages to be turned into usable Registry metacards and for those metacards to be turned back into CSW messages.
```

Comment:
Replace this definition with "the Registry Store provides a means of configuring connections to other registries and controlling update behavior."